packages/p2p/src/stores/my-profile-store.js

```diff
@@ -395,6 +395,7 @@ export default class MyProfileStore extends BaseStore {
                 } else {
                     this.setSearchResults([]);
                 }
+                this.root_store.buy_sell_store.setIsFilterModalLoading(false);
```

Comment:
do we need this? we already have it on line 400

Reply:
Nope this is redundant, I'll remove it asap